exercises/palindrome-products/README.md

```diff
@@ -9,27 +9,28 @@ Given a range of numbers, find the largest and smallest palindromes which
 are products of numbers within that range.
 
 Your solution should return the largest and smallest palindromes, along with the
-factors of each within the range. If the largest or smallest palindrome has more 
+factors of each within the range. If the largest or smallest palindrome has more
 than one pair of factors within the range, then return all the pairs.
 
-Not all ranges of factors can produce a Palindrome, and ranges must have their 
```

Comment:
This text is specific to ocaml and I think helpful to the exercise. Please don't delete.

Reply:
@stevejb71 I'll fork the exercise so that this doesn't get the changes from the upstream spec.